### PR TITLE
Add and update the documentation 

### PR DESCRIPTION
**Pull request by lucascanero**
New Documentation:
 - Create New Project
 - Add New Site

Updates:
  - README.md - Add "Project Template" section with a redirection link to [/lucascanero/terrabutler-aws-template](https://github.com/lucascanero/terrabutler-aws-template)
  - docs/requirements.md - Add Python3 as a requirement.
  - mkdocs.yml - Update file to include the new documention.

### CODE REVIEW

File: docs/create-project.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Create a new project
+
+Before proceeding make sure that you have followed the [Installation](installation.md). 
+
+## Configure direnv
+
+For direnv to work properly it needs to be hooked into the shell.
+
+Make sure that you have hooked direnv into your bash shell before proceeding.
```
Comment (MiguelNdeCarvalho) [inline docs/create-project.md:9]: I think it's better if we point it to the DirENV docs itself where it shows the config for other shells not only for Bash. https://direnv.net/docs/hook.html

File: docs/create-project.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+## Configure direnv
+
+For direnv to work properly it needs to be hooked into the shell.
+
+Make sure that you have hooked direnv into your bash shell before proceeding. 
+
+If not, add the following line at the end of your `~/.bashrc` file:
+
+```
+eval "$(direnv hook bash)"
```
Comment (MiguelNdeCarvalho) [inline docs/create-project.md:14]: After adding the docs above, this could be removed

File: docs/new-site_.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+# Add a new site_
```
Comment (MiguelNdeCarvalho) [inline docs/new-site_.md:1]: Remove the `_` from the tile and from the file name

File: docs/new-site_.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+In the project folder root, create a new diretory called `site_<site-name>`:
+
+```shell
+$ mkdir site_<site-name>
+```
+???+ tip
+  Replace `<site-name>`  with your new site name!
+
+## Configure the new `site_`
```
Comment (MiguelNdeCarvalho) [inline docs/new-site_.md:13]: Again remove the `_`

File: docs/new-site_.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+```shell
+$ ln -s ../globals/data.tf ./data_global.tf
+```
+```shell
+$ ln -s ../globals/locals.tf ./locals_globals.tf
+```
+```shell
+$ ln -s ../globals/variables.tf ./variables_globals.tf
+```
+### Perform an init in `site_`
```
Comment (MiguelNdeCarvalho) [inline docs/new-site_.md:83]: Same here, remove the `_`

File: docs/requirements.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ Before you start using you will need to have some tools installed.
 
 - [direnv](https://direnv.net/)
 - [tfenv](https://github.com/tfutils/tfenv)
+- [python3](https://www.python.org/)
```
Comment (MiguelNdeCarvalho) [inline docs/requirements.md:7]: Not sure if this is a requirement as we are shipping the binaries which should work without Python present in the machine. But this should be confirmed before removed

File: docs/requirements.md [hunk 7]
```diff
@@ -13,6 +14,7 @@ In the list below you have the install instruction for each tool:
 
 - [direnv](https://direnv.net/docs/installation.html)
 - [tfenv](https://github.com/tfutils/tfenv#installation)
+- [python3](https://wiki.python.org/moin/BeginnersGuide/Download)
```
Comment (MiguelNdeCarvalho) [inline docs/requirements.md:17]: Same as last comment

File: mkdocs.yml [hunk 8]
```diff
@@ -56,3 +56,5 @@ nav:
   - Requirements: requirements.md
   - Installation: installation.md
   - Basic Usage: usage.md
+  - Create New Project: create-project.md
+  - Add New site_ : new-site_.md
```
Comment (MiguelNdeCarvalho) [inline mkdocs.yml:60]: This should look a bit better `- Add New Site : new-site.md`

**Review by MiguelNdeCarvalho — CHANGES_REQUESTED** [review]
Really thanks for your great contribution, I have made some reviews, feel free to take a look on them and see if you agree with them

File: docs/create-project.md (hunk 1 above)
Comment (lucascanero) [reply docs/create-project.md:9]: Request done! Now pointing to https://direnv.net/docs/hook.html

File: docs/create-project.md (hunk 2 above)
Comment (lucascanero) [reply docs/create-project.md]: Deleted!

File: docs/new-site_.md (hunk 3 above)
Comment (lucascanero) [reply docs/new-site_.md]: Deleted!

File: docs/new-site_.md (hunk 5 above)
Comment (lucascanero) [reply docs/new-site_.md]: Deleted!

File: mkdocs.yml (hunk 8 above)
Comment (lucascanero) [reply mkdocs.yml]: Changed!

File: docs/requirements.md (hunk 6 above)
Comment (MiguelNdeCarvalho) [reply docs/requirements.md:7]: Thanks @lucascanero

**Comment by MiguelNdeCarvalho** [comment]
Really thanks @lucascanero, so the only one missing is the Python requirement and then the PR should be ready to merge

**Comment by MiguelNdeCarvalho** [comment]
So @lucascanero I have tested `Terrabutler` inside of `Ubuntu` and `Debian` docker image and it worked like a charm. So it shouldn't be needed as a requirement right? Thanks

**Comment by lucascanero** [comment]
Yes, @MiguelNdeCarvalho you are right! When terrabutler is installed all python binaries are present. Another thing that i notice is that terrabutler only works on Intel based processors. In Arm and Arm64 is not possible to install terrabutler. I dont know if you want to reference that information on the documention.

**Review by lucascanero — COMMENTED** [review]
All request reviewed and changed!

**Review by lucascanero — COMMENTED** [review]
add "only intel based processor architecture are compatible" warning to documentation

**Comment by MiguelNdeCarvalho** [comment]
Everything looks good to me know. I think in the future we can add support to ARM and ARM64 architecture. Can I merge the PR then?

**Review by MiguelNdeCarvalho — APPROVED** [review]

**Comment by MiguelNdeCarvalho** [comment]
Thanks for everything @lucascanero!

**Comment by lucascanero** [comment]
@MiguelNdeCarvalho You welcome!

**Pull request merged**